4631.bugfix.rst

```diff
@@ -0,0 +1 @@
+Don't rewrite assertion when __getattr__ is broken
```

Comment:
Small nitpick:

```rst
Don't rewrite assertion when ``__getattr__`` is broken.
```

Reply:
Thanks! Just realized i also did not put this file in the `changelog` folder.